### ISSUE TITLE
cannot join a new shared wallet - exits after QR code scan

### ISSUE THREAD

**Issue by mpolci** [bitpay/copay#3552]
I have copay v1.5.2 on two different android devices. 

I started creating a new 2 of 2 shared wallet on the testnet network and it showed me the secret and the QR code.

On the second device I started joining the wallet, I compiled the nickname and started scanning the QR code.
After the QR code scan it returns immediately at the home screen without making the join.

This is the session log:
```
Copay Session Logs
Be careful, this could contain sensitive private data

Starting Copay v1.5.2 #c553fbc
Client: isCordova: true isChromeApp: null isNodeWebkit: undefined
Navigator: Mozilla/5.0 (Linux; Android 5.0.2; LG-D802 Build/LRX22G; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/46.0.2490.76 Mobile Safari/537.36
Using file storage: true
Setting default language: it
Got main dir: file:///data/data/com.bitpay.copay/files/
Read:  true
Read:  {"version":"1.0.0","createdOn":1429354570250,"credentials":[{"network":"testnet","xPrivKey":"**********","xPubKey":"tpubD97G4ZrMtPxqFgCfMzqoMthdjE8z9yLmVpzQvPtQQKYCb7UKTnWmodUS64wputV6bxCwfva1ekAi5xY2aGe1HEs8biA6AcfmwgTpn3DS3SR","requestPrivKey":"**********","requestPubKey":"029349237fb679d88197151b41a2d00bb1651322edf53fb51dfb1e350ed955c0bf","copayerId":"b4cb12fcdf03d25cfe75d936cc297071c63ec6b3a081268b7229b229aabebaff","publicKeyRing":[{"xPubKey":"tpubD97G4ZrMtPxqFgCfMzqoMthdjE8z9yLmVpzQvPtQQKYCb7UKTnWmodUS64wputV6bxCwfva1ekAi5xY2aGe1HEs8biA6AcfmwgTpn3DS3SR","requestPubKey":"029349237fb679d88197151b41a2d00bb1651322edf53fb51dfb1e350ed955c0bf","isTemporaryRequestKey":false,"copayerName":"Telefono"},{"xPubKey":"tpubD9gj9GKhP9eyZKsHoyHunQfWfb1MuQCe1sGCX6qNJLnzLHktAEqJSBkEj8mH16xHfz4JF7i5NMyAvVMtQjVbZmPDBx2jJKjaGe8s3BdM3Ay","requestPub...
Profile is not encrypted
Profile read
Read:  {"limits":{"totalCopayers":6,"mPlusN":100},"bws":{"url":"https://bws.bitpay.com/bws/api"},"wallet":{"requiredCopayers":2,"totalCopayers":3,"spendUnconfirmed":true,"reconnectDelay":5000,"idleDurationMin":4,"settings":{"unitName":"bits","unitToSatoshi":100,"unitDecimals":2,"unitCode":"bit","alternativeName":"Eurozone Euro","alternativeIsoCode":"EUR"}},"glidera":{"enabled":true,"testnet":false},"rates":{"url":"https://insight.bitpay.com:443/api/rates"},"insight":{"testnet":{"url":"https://test-insight.bitpay.com:443","transports":["polling"]},"livenet":{"url":"https://insight.bitpay.com:443","transports":["polling"]}},"passphraseConfig":{"iterations":5000,"storageSalt":"mjuBtGybi/4="},"colorFor":{"3d2a0f59-c3e1-440e-9618-52b088795b6c":"#FADA58"},"bwsFor":{"5a731c7d-1cb0-42ba-9498-116aff2cef14":"https://bws.bitpay.com/bws/api"}}
Preferences read: {"limits":{"totalCopayers":6,"mPlusN":100},"bws":{"url":"https://bws.bitpay.com/bws/api"},"wallet":{"requiredCopayers":2,"totalCopayers":3,"spendUnconfirmed":true,"reconnectDelay":5000,"idleDurationMin":4,"settings":{"unitName":"bits","unitToSatoshi":100,"unitDecimals":2,"unitCode":"bit","alternativeName":"Eurozone Euro","alternativeIsoCode":"EUR"}},"glidera":{"enabled":true,"testnet":false},"rates":{"url":"https://insight.bitpay.com:443/api/rates"},"insight":{"testnet":{"url":"https://test-insight.bitpay.com:443","transports":["polling"]},"livenet":{"url":"https://insight.bitpay.com:443","transports":["polling"]}},"passphraseConfig":{"iterations":5000,"storageSalt":"mjuBtGybi/4="},"colorFor":{"3d2a0f59-c3e1-440e-9618-52b088795b6c":"#FADA58"},"bwsFor":{"5a731c7d-1cb0-42ba-9498-116aff2cef14":"https://bws.bitpay.com/bws/api"}}
Preferences read
Set focus: null
Profile loaded ... Starting UX.
Read:  true
Read:  1433483797691
onGoingProcess openingWallet true
onGoingProcess openingWallet null
Wallet Opened
Updating Transaction History
Read:  2MwYNefVyXxXAtMtXpe3DbCLs9Bm4dz7hg8
onGoingProcess updatingStatus true
Updating Status: Testnet null
Binding touchstart elements...
Read:  [{"txid":"f7ab175815c34f60f5b502dae1bf4ef58dcdbbd6269a0d144c1de5cd606d1dc3","action":"moved","amount":485668,"fees":10000,"time":1434312751,"addressTo":"N/A","outputs":[{"amount":485668,"address":"2NGU2X1aNbQnz2Git35s6NT9NREqZZuxBMK","message":null,"encryptedMessage":null}],"confirmations":140743,"proposalId":"01434312657962e0519908-2522-4494-8026-a43d3660cec6","proposalType":"simple","creatorName":"Telefono","message":null,"actions":[{"createdOn":1434312658,"type":"accept","copayerId":"b4cb12fcdf03d25cfe75d936cc297071c63ec6b3a081268b7229b229aabebaff","copayerName":"Telefono","comment":""},{"createdOn":1434312710,"type":"accept","copayerId":"5a600d2df6a66b2e20269e4c490bccbdb3e6ef05a4a5ea832e9e66f89d363ebd","copayerName":"tab","comment":""}],"encryptedMessage":null,"hasUnconfirmedInputs":false,"amountStr":"4,857 bits","alternativeAmountStr":"1.65 EUR","feeStr":"100 bits","safeConfirmed":"6+"},{"txid":"355df1ac4316f37d2070249f8d6bbb5ad342c822f346ab0e5527cb5ddc3a3b5b","action":"receive...
onGoingProcess scanning undefined
Status update. hash:636273 Try:0
onGoingProcess updatingStatus null
Wallet Status: {"wallet":{"version":"1.0.0","createdOn":1432653167,"id":"3d2a0f59-c3e1-440e-9618-52b088795b6c","name":"Testnet","m":2,"n":2,"status":"complete","copayers":[{"version":2,"createdOn":1432653167,"id":"b4cb12fcdf03d25cfe75d936cc297071c63ec6b3a081268b7229b229aabebaff","name":"Telefono","requestPubKeys":[{"key":"029349237fb679d88197151b41a2d00bb1651322edf53fb51dfb1e350ed955c0bf","signature":"3045022100a5b495a252f443c1bafd4a2b58b98e8093a851a14b6edd0a3be3a9eca662481502200123aecf89cac415702b30c05ceac5ec7af6263bdf519f1cfeef37b95af55ad5"}]},{"version":2,"createdOn":1432653572,"id":"5a600d2df6a66b2e20269e4c490bccbdb3e6ef05a4a5ea832e9e66f89d363ebd","name":"tab","requestPubKeys":[{"key":"03994d53a88b2c5000babd9dd80086ea1defc4bd35908de41a6d9af43aa0f78a03","signature":"304502210081bbcdcaa83fbb5d8e26569c0144d13f43c17c911fd52511f0348f77b3f56c4e0220317f7fc84d6db2966e411a9c43c8435c963c5d354f45696587d7da47aa7f08f6"}]}],"network":"testnet","derivationStrategy":"BIP45","addressType":"P2SH","scanStatus":n...
Syncing TXs. Got:0 Skip:6  EndingTxid: f7ab175815c34f60f5b502dae1bf4ef58dcdbbd6269a0d144c1de5cd606d1dc3  Continue: null
Finish Sync: New Txs: 0
Tx History synced. Total Txs: 5
Read:  2MwYNefVyXxXAtMtXpe3DbCLs9Bm4dz7hg8
Writing: txsHistory-3d2a0f59-c3e1-440e-9618-52b088795b6c [{"txid":"f7ab175815c34f60f5b502dae1bf4ef58dcdbbd6269a0d144c1de5cd606d1dc3","action":"moved","amount":485668,"fees":10000,"time":1434312751,"addressTo":"N/A","outputs":[{"amount":485668,"address":"2NGU2X1aNbQnz2Git35s6NT9NREqZZuxBMK","message":null,"encryptedMessage":null}],"confirmations":140743,"proposalId":"01434312657962e0519908-2522-4494-8026-a43d3660cec6","proposalType":"simple","creatorName":"Telefono","message":null,"actions":[{"createdOn":1434312658,"type":"accept","copayerId":"b4cb12fcdf03d25cfe75d936cc297071c63ec6b3a081268b7229b229aabebaff","copayerName":"Telefono","comment":""},{"createdOn":1434312710,"type":"accept","copayerId":"5a600d2df6a66b2e20269e4c490bccbdb3e6ef05a4a5ea832e9e66f89d363ebd","copayerName":"tab","comment":""}],"encryptedMessage":null,"hasUnconfirmedInputs":false,"amountStr":"4,857 bits","alternativeAmountStr":"1.65 EUR","feeStr":"100 bits","safeConfirmed":"6+"},{"txid":"355df1ac4316f37d2070249f8d6bbb5ad342c822f346ab0e5527cb5ddc3a3b5b","action":"receive...
Dynamic fee: normal 5008 SAT
Cannot read property 'secret' of undefined
Binding touchstart elements...
Read:  2MwYNefVyXxXAtMtXpe3DbCLs9Bm4dz7hg8

```

**Comment by matiu** [comment]
Thanks, we where able to reproduce it. It is Android specific, we will send a fix as soon as possible and update this thread.

**Comment by matiu** [comment]
OK, the fix is already merged and deployed to the app store (1.5.3) it should be available to upgrade in a couple of hours. thanks a lot.
